src/rad/inspector/wxfbadvprops.cpp

```diff
@@ -578,10 +594,33 @@ wxVariant wxFBBitmapProperty::ChildChanged(wxVariant& thisValue, const int child
 
                     break;
                 }
-                // 'Load From Art Provider'
-                case 4:
+				// 'Load From XRC'
+				case 4:
+				{
+					if (prevSrc != 4)
+					{
+						for (unsigned int i = 1; i<count; i++)
+						{
+							wxPGProperty *p = Item(i);
+							if (p)
+							{
+								wxLogDebug(wxT("wxFBBP::ChildChanged: Removing:%s"), p->GetLabel().c_str());
+								GetGrid()->DeleteProperty(p);
+							}
+						}
+						bp->AppendChild(bp->CreatePropertyXrcName());
+					}
+
+					if (childVals.GetCount() == 2)
+						newVal = childVals.Item(0) + wxT("; ") + childVals.Item(1);
+					else if (childVals.GetCount() > 1)
+						newVal = childVals.Item(0) + wxT("; ");
+					break;
+				}
+				// 'Load From Art Provider'
+                case 5:
```

Comment:
Dealing with formatting is such a tedious work here, i have to disable pretty much all auto formatting or Visual Studio reformats half the screen because it wants to correct the surroundings. Even worse when i let it pick up the clang-format file, then it reformats the whole file up to the changed line. Plus the format the clang-format definition produces doesn't look like the format you want to have.

Reply:
> Plus the format the clang-format definition produces doesn't look like the format you want to have.

The `.clang-format` file uses tabs and has `IndentCaseLabels: true`. What difference do you mean?

Reply:
The braces are java-style, the opening brace is on the same line, but you want it on the next line.